src/ensembl/src/root/Root.tsx

```diff
@@ -9,34 +13,22 @@ import privacyBannerService from '../shared/components/privacy-banner/privacy-ba
 import ErrorBoundary from 'src/shared/components/error-boundary/ErrorBoundary';
 import { GeneralErrorScreen } from 'src/shared/components/error-screen';
 
-import { updateBreakpointWidth } from '../global/globalActions';
-import { getBreakpointWidth } from '../global/globalSelectors';
-import { RootState } from '../store';
-import { BreakpointWidth } from '../global/globalConfig';
-import { getBreakpoint } from '../global/globalHelper';
-
 import styles from './Root.scss';
 
-type StateProps = {
-  breakpointWidth: BreakpointWidth;
-};
-
-type DispatchProps = {
+type Props = {
   updateBreakpointWidth: (breakpointWidth: BreakpointWidth) => void;
 };
 
-type OwnProps = {};
-
-type RootProps = StateProps & DispatchProps & OwnProps;
-
-export const Root: FunctionComponent<RootProps> = (props: RootProps) => {
-  const [ref, width] = useResizeObserver();
-  const currentBreakpoint: BreakpointWidth = getBreakpoint(width);
+export const Root = (props: Props) => {
+  const elementRef = useRef<HTMLDivElement>(null);
+  // TODO: this should be properly done via window.matchMedia() rather than
+  // via ResizeObserver; will refactor in subsequent PR
```

Comment:
see description of [this PR](https://github.com/Ensembl/ensembl-client/pull/215)